packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 4.2.2 - Feb 7, 2023
+
+Incremental version for Teams Toolkit with multiple bugs fixes and enhancements.
+
+Enhancements:
+
+- Updated pre-requisite checker, applicable app templates and samples to support Node.js 18.
+- Updated comments on Notification bot template for how to send notifications to an individual person.
+- Updated TeamsFx-React SDK to support React 17.
+- Updated TeamsFx SDK, relevant bot app templates and samples to use CloudAdapter.
```

Comment:
Would it be better to add link to *CloudAdapter* (e.g., https://learn.microsoft.com/javascript/api/botbuilder/cloudadapter), or call out it's Bot Framework SDK's CloudAdapter?

Reply:
Updated.